src/disco/components/Addon.js

```diff
@@ -116,43 +112,27 @@ export class AddonBase extends React.Component {
   }
 
   getThemeImage() {
-    const { getBrowserThemeData, i18n, name, previewURL } = this.props;
+    const { i18n, name, previewURL } = this.props;
     if (this.props.type === ADDON_TYPE_THEME) {
       /* eslint-disable jsx-a11y/href-no-hash, jsx-a11y/anchor-is-valid */
```

Comment:
You may not need this comment anymore either, having removed much of the following code. I'd try removing it and see if eslint complains.